server/aspnetcore/TicketHub/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning",
+      "Styra.Opa.AspNetCore": "Trace"

```

Comment:
This line *in particular* is required if you want to see the Trace-level logs from the framework. Absolutely crucial for a good debugging experience. :sweat_smile: